nyansat/host/shell/__main__.py

```diff
@@ -660,18 +660,7 @@ def do_elevation(self, args):
         parsed_args = parse_cli_args(args, 'elevation', 1, arg_properties)
         el, = parsed_args
         self.client.elevation(el)
-        if self._is_open():
```

Comment:
I'm assuming this is in the middle of a refactor, hence all of this logic is deleted